Okta/OktaAuth.swift

```diff
@@ -60,6 +62,46 @@ public struct OktaAuthorization {
     func passwordFlow(_ config: [String: String], credentials: [String: String]?, _ view: UIViewController) -> Promise<OktaTokenManager> {
         return buildAndPerformTokenRequest(config, additionalParams: credentials)
     }
+    
+    func logout(_ config: [String: String], idToken: String?, view: UIViewController) -> Promise<Void> {
+        return Promise<Void>(in: .background, { resolve, reject, _ in
+            guard let idToken = idToken else {
+                return reject(OktaError.MissingUserIdToken)
+            }
+
+            guard let issuer = config["issuer"],
+                let logoutRedirectUri = config["logoutRedirectUri"] else {
+                    return reject(OktaError.MissingConfigurationValues)
+            }
+            
+            self.getMetadataConfig(URL(string: issuer))
+                .then { oidConfig in
+
+                    let request = OIDEndSessionRequest(
+                            configuration: oidConfig,
+                            idTokenHint: idToken,
+                            postLogoutRedirectURL: URL(string: logoutRedirectUri)!,
```

Comment:
nit: Move the URL cast to the `guard` statement above, so we don't need to force unwrap.